src/utils/typography.js

```diff
@@ -0,0 +1,10 @@
+import './font.css'
```

Comment:
is it necessary to import the css file

Reply:
How else are we going to include the local font in our code?